src/Phinx/Migration/Manager.php

```diff
@@ -368,61 +391,89 @@ public function executeSeed($name, SeedInterface $seed)
      * Rollback an environment to the specified version.
      *
      * @param string $environment Environment
-     * @param int $version
+     * @param int $target
      * @param bool $force
+     * @param bool $targetMustMatchVersionName
      * @return void
      */
-    public function rollback($environment, $version = null, $force = false)
+    public function rollback($environment, $target = null, $force = false, $targetMustMatchVersionName = true)
```

Comment:
can do. The problem was that `version` sometimes refered to the creation time and sometimes to the array with all the version data (creation time, start time etc). But again for consistency, renaming